docs/articles/expensify-classic/expensify-card/Expensify-Card-Perks.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Expensify Card Perks
+description: Get the most out of your Expensify Card with exclusive perks!
+---
+
+
+# Overview
+The Expensify Card is packed with perks, both native to our Card program and through exclusive discounts with partnering solutions. The Expensify Card’s primary perks include: 
+- Access to our premiere Expensify Lounge (with more locations coming soon)
+- Swipe to Win, where every swipe has a chance to win fun personalized gifts for you and your closest friends and family members
+- And unbeatable cash back incentive with each swipe
+Below, we’ll cover all of our exclusive offers in more detail and how to claim discounts with our partners.
+
+# Expensify Card Perks
+
+## Access to the Expensify Lounge
+Our [world-class lounge](https://use.expensify.com/lounge) is now open for Expensify members and guests to enjoy! 
+
+We invite you to visit our sleek San Francisco lounge, where sweeping city views provide the perfect backdrop for a morning coffee to start your day. 
+
+Enjoy complimentary cocktails and snacks in a vibrant atmosphere with blazing-fast WiFi. Whether you want a place to focus on work, socialize with other members, or simply kick back and relax – our lounge is ready and waiting to welcome you.
+
+You can sign up for free [here](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Fuse.expensify.com%2F) if you’re not an Expensify member. If you have any questions, reach out to concierge@expensify.com and [check](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Fuse.expensify.com%2Flounges) this out for more info.
```

Comment:
This link goes through community.expensify.com just to re-route to use.expensify.com. Since we're trying to deprecate community, let's update this link accordingly:

```suggestion
You can sign up for free [here](https://use.expensify.com) if you’re not an Expensify member. If you have any questions, reach out to concierge@expensify.com and [check this out](https://use.expensify.com/lounge) for more info.
```